lib/assessment.js

```diff
@@ -8,38 +8,74 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue) {
+  if previousValue < currentValue{
```

Comment:
Missing parentheses (`(`, `)`) around the condition (`previousValue < currentValue`) is a syntax error.